src/pages/Report.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Outlet } from "react-router-dom";
+import { Container } from "../components/styles/Container.styled";
+import Header from "../components/Header";
+import mailIcon from "../assets/imgs/mail.svg";
+import importantIcon from "../assets/imgs/important_vector.png";
+import { SimpleFooter } from "../components/SimpleFooter";
+
+const Report = () => {
+  const [message, setMessage] = useState("");
+
+  function handleMessage(e: any): void {
```

Comment:
Para ganho de performance, transformar essa `function` em arrow function. 
> Não precisa inserir `void` como retorno da função.
> Adicionar `ChangeEvent` type e importa-lo do `React`
```suggestion
  const handleMessage = (e: ChangeEvent<HTMLTextAreaElement>) => {
```

Reply:
@CarlosJunioor atenção a esses comentários.

Reply:
ta feito

Reply:
Good job!